### ISSUE TITLE
A11y_Power Virtual Agents_Chat window_ScreenReader:When narrator focus is no cross button it is announcing unwanted information .

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#4561]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Screen reader users will get wrong information while navigating in scan mode.

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### What is the public URL for the website?

https://web.powerva.microsoft.com/environments/839eace6-59ab-4243-97ec-a5b8fcc104e4/bots/107725f1-d827-41a4-8375-c91d1a97affc

### How to reproduce the issue?

1. Launch the application using URL: [Home - Test | Power Virtual Agents (microsoft.com)](https://web.powerva.microsoft.com/environments/839eace6-59ab-4243-97ec-a5b8fcc104e4/bots/107725f1-d827-41a4-8375-c91d1a97affc)
1. Tab till chat box and press enter to open it.
1. Start Narrator.
1. Tab till cross button and use down arrow,

### What do you expect?

when using down arrow key to navigate narrator should announce chat window details showing up in chat box.

### What actually happened?

When narrator focus is no cross button it is announcing unwanted information .

Not observed with NVDA

### MAS reference

[MAS 1.3.2 – Meaningful Sequence](https://aka.ms/MAS1.3.2)

### Additional context

Need investigation.

This is partially fixed in 4.15.5, however, it still have an issue:

1. Turn on Narrator
1. Press <kbd>TAB</kbd> to focus on the keyboard help screen
1. Press down arrow key
1. Expect: It should read the title of the dialog "Keyboard controls"
1. Actual: It read the subtitle of the dialog "Chat window", skipped the title

https://msazure.visualstudio.com/CCI/_workitems/edit/15627865

**Comment by compulim** [comment]
I am testing it on 4.15.6 with latest Narrator and it looks good.

https://user-images.githubusercontent.com/1622400/211699155-88654019-848f-45ee-9bc2-be98c590a183.mp4

**Comment by compulim** [comment]
Things to do:
- Move `<button>` into `<header>` and possibly put it before `<h2>Keyboard controls</h2>`
- Add `aria-labelledby` to `<div role="dialog">` and IDREF it to `<h2>Keyboard controls</h2>`

With Windows Narrator, it should narrate "Keyboard controls dialog, banner header close button". Then, when <kbd>DOWN</kbd> arrow key is pressed, it will narrate "Heading level 2 keyboard controls".

**Comment by compulim** [comment]
After adding `aria-labelledby` and reordering `<button>` before `<h2>`.

https://user-images.githubusercontent.com/1622400/211921369-147600df-e1c0-4b22-86c7-a25a4c731d5c.mp4
